tests/testthat/_snaps/list_distributions.md

```diff
@@ -228,79 +228,225 @@
-      23 WHO_Ebola_Response_Team 2015          49
-      24 WHO_Ebola_Response_Team 2015         957
-      25 WHO_Ebola_Response_Team 2015         792
-      26  WHO_Ebola_ResponseTeam 2015        1798
-      27            Lessler_etal 2009          13
-      28              Ghani_etal 2009          16
-      29            Lessler_etal 2009         151
-      30            Lessler_etal 2009          90
-      31            Lessler_etal 2009          78
-      32            Lessler_etal 2009         124
-      33          Nishiura_Inaba 2011          72
-      34          Nishiura_Inaba 2011          72
-      35              Reich_etal 2009         151
-      36              Reich_etal 2009         151
-      37              Tuite_etal 2010         316
-      38          Virlogeux_etal 2015         229
-      39          Virlogeux_etal 2015         229
-      40          Virlogeux_etal 2016         395
-      41          Virlogeux_etal 2016         173
-      42          Virlogeux_etal 2016         222
-      43            Rudolph_etal 2014           6
-      44                  Pavlin 2014          76
-      45                  Pavlin 2014          18
-      46            Lessler_etal 2009          55
-      47             Assiri_etal 2013          23
-      48            Cowling_etal 2015         166
-      49           Charniga_etal 2022          22
-      50            Guzetta_etal 2022         255
-      51           Madewell_etal 2022          35
-      52           Madewell_etal 2022          36
-      53              Miura_etal 2022          18
-      54               Wang_etal 2022          16
-      55               Wang_etal 2022          27
-      56               Wang_etal 2022         114
-      57                Wei_etal 2022          NA
-      58                Wei_etal 2022          NA
-      59                Wei_etal 2022          NA
-      60            Lessler_etal 2009          11
-      61            Lessler_etal 2009          28
-      62            Rudolph_etal 2014          23
-      63            Lessler_etal 2009          24
-      64              Reich_etal 2009          24
-      65              Reich_etal 2009          24
-      66            Lessler_etal 2009         157
-      67            Rudolph_etal 2014          18
-      68            Rudolph_etal 2014           8
-      69            Rudolph_etal 2014           6
-      70            Rudolph_etal 2014          91
-      71            Rudolph_etal 2014          80
-      72            Lessler_etal 2016          25
+                          author
+      1             Lessler_etal
+      2             Rudolph_etal
+      3               Alene_etal
+      4                 Bui_etal
+      5               Elias_etal
+      6               Lauer_etal
+      7               Lauer_etal
+      8               Lauer_etal
+      9               Lauer_etal
+      10             Linton_etal
+      11             Linton_etal
+      12             Linton_etal
```

Comment:
Tangential to this PR but are there any plans to list all authors instead of using `etal`? It would be especially useful if authors want to use the bibtex citation string directly in their manuscript or reference manager.

Reply:
Yes, the plan is to store the all bibliographic information. I left it as just the shortened author list for this PR to get the changes into main quicker. I think that the tabular (csv) format is not conducive to a list of names (e.g. one cell for all authors which are semicolon separated perhaps, because one cell per author would not work). Instead using JSON where the authors could be listed within the bibliography of each entry might be better. What do you think to changing from csv to a more structured data format? I don't foresee it being more difficult to maintain or expand a JSON database than csv.